src/SliderProvider/index.tsx

```diff
@@ -94,6 +94,10 @@ const SliderProvider: React.FC<SliderProviderProps> = (props) => {
     slides: [],
   });
 
+  const memoizedSlideWidth = useMemo(() => {
+    return `${(slidesToShow > 1 ? 1 / slidesToShow : slidesToShow) * 100}%`
+  }, [slidesToShow]);
+
```

Comment:
No need to memoize. We can just write `const slideWidth = ...`

Reply:
I think we may want to keep the slider widths uninitialized, see my comment here: https://github.com/faceless-ui/slider/issues/17#issuecomment-1518064005

Reply:
I don't see the benefit of waiting a render cycle to provide the _known information_ via context?